server/build.gradle.kts

```diff
@@ -101,6 +101,7 @@ val deltaTest = tasks.register<Test>("deltaTest") {
 }
 tasks.check {
     dependsOn(deltaTest)
+    finalizedBy(tasks.jacocoTestReport)
```

Comment:
how can I get a coverage report?
Can you add a threshold and make the CI build fail if the threshold Is not met?
can GitHub visualise the result in the PR?

Reply:
I did not want to add rules yet, as we have 60% coverage for now, it would fail a lot;
maybe something very loose like 50% coverage is fine for now?

the jacocoTestReport should print the location of the index.html, but maybe I removed it when trying out 
I will check the github visualization

Reply:
I would say, let's evaluate the current code coverage (i.e. 60%) and let's put that as the threshold. If any PR degrades the threshold then we can decide to:
1. write more tests
2. lower the threshold

In this way we can keep the bar "high"